.github/workflows/application-signals-e2e-test.yml

```diff
@@ -46,7 +46,7 @@ jobs:
   # DEFAULT SETTING: {Python Version}, EC2, AMD64, AL2
   #
 
-  default-v8-amd64:
+  default-v3-8-amd64:
```

Comment:
can we do `default-py38-amd64` instead? Using `py{version without dots}` seems like a general practice ([otel](https://github.com/open-telemetry/opentelemetry-python/blob/main/tox.ini#L9), [xray](https://github.com/aws/aws-xray-sdk-python/actions/runs/12711671416))